src/metabase/driver/duckdb.clj

```diff
@@ -329,6 +337,7 @@
 (defmethod driver/describe-table :duckdb
   [driver database {table_name :name, schema :schema}]
   (let [database_file (get (get database :details) :database_file)
+        database_file (first (str/split database_file #"\?"))  ;; remove additional options in connection string
```

Comment:
I vaguely feel like `database_file (first (str/split database_file #"\?")` wants to be a helper function